agent-bootstrap/src/main/java/io/opentelemetry/auto/bootstrap/instrumentation/decorator/BaseDecorator.java

```diff
@@ -83,13 +84,13 @@ public Span onPeerConnection(final Span span, final InetSocketAddress remoteConn
       } else {
         // Failed DNS lookup, the host string is the name.
         final String hostString = remoteConnection.getHostString();
-        span.setAttribute(MoreTags.NET_PEER_NAME, hostString);
+        span.setAttribute(SemanticAttributes.NET_PEER_NAME.key(), hostString);
```

Comment:
we should use `SemanticAttributes` like this everwhere, we can do that in separate PR though, probably good to merge this PR quickly to avoid conflicts

```suggestion
        SemanticAttributes.NET_PEER_NAME.set(span, hostString);
```

Reply:
@trask if PR will be merged I will work on another one to use `SemanticAttributes.NET_PEER_NAME.set(span, hostString);`

Reply:
Yeah if I recall correctly we're waiting for the SDK version bump to have the form that supports setting into a builder

Reply:
anybody works on SDK version bump? looking for it.

Reply:
I'm guessing @iNikem is based on https://gitter.im/open-telemetry/opentelemetry-java-instrumentation?at=5f034b32fa0c9221fc818e08